.github/workflows/push_to_docker_hub.yml

```diff
@@ -7,106 +7,26 @@ on:
       - 'develop'
   push:
     tags:
-      # Just create image on pushing a tag
+      # Create images on pushing a tag
       - '*'
```

Comment:
I think it would be good to use a pattern that matches our release tags only, as we do in https://github.com/sagemath/sage/blob/develop/.github/workflows/dist.yml#L6

(I frequently push other tags to my repository for triggering CI runs, and currently this triggers a failing Docker Hub workflow.)

Reply:
> I think it would be good to use a pattern that matches our release tags only, as we do in https://github.com/sagemath/sage/blob/develop/.github/workflows/dist.yml#L6
> 
> (I frequently push other tags to my repository for triggering CI runs, and currently this triggers a failing Docker Hub workflow.)

Done!